GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -194,10 +208,24 @@ public virtual void ApproveCredentials(string repoUrl, string username, string p
 
             string stdinConfig = sb.ToString();
 
-            this.InvokeGitOutsideEnlistment(
+            Result result = this.InvokeGitOutsideEnlistment(
                 "credential approve",
                 stdin => stdin.Write(stdinConfig),
                 null);
+
+            if (result.ExitCodeIsFailure)
+            {
+                EventMetadata errorData = new EventMetadata();

```

Comment:
Same comment here, this `EventMetadata` is not needed if we're not adding anything to it.